src/GlobalStyle.js

```diff
@@ -0,0 +1,13 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+ * {
+   box-sizing: border-box
+ }
+ body {
+   margin: 0;
+   font-family: 'Roboto', sans-serif;
+   font-size: 112.5%;
+   line-height: 1.4;
+ }
+`
```

Comment:
I would call the file in plural: GlobalStyles
probably just my personal opinion